WtWSMS/localization/english/culture/traditions/BP_cultural_languages_l_english.yml

```diff
@@ -39,7 +38,7 @@
  language_gallic_romance_name: "Gallic Romance"
  language_gandhari_name: "Gandhari"
  language_gaulish_name: "Gaulish"
- language_germano_african_romance_name: "Arréyanu"
+ language_germano_african_romance_name: "Germano-Arréyanu"
```

Comment:
?

Reply:
Last I checked we wanted a variant of African Romance with Germanic superstrate; we have it for the other major Romance groups with Germanic kingdoms (e.g., Romano-Gallic, Romano-Hispanic, Italian Romance). There's just no good name for it.

Reply:
Ok, I would probably rather go with African (from African Romance)

Reply:
Renamed "Germano-African Romance".